examples/benchmark_evaluator.py

```diff
@@ -0,0 +1,189 @@
+from evaluate_existing_episode import run_async_server_in_batch_aevaluate
+from sotopia.database import (
+    human_annotation_to_episodelog,
+    AnnotationForEpisode,
+    EpisodeLog,
+)
+from typer import Typer
+import numpy as np
+import pandas as pd
+import scipy.stats as stats
+
+from sotopia.database.serialization import get_rewards_from_episode
+
+app = Typer()
+
+target_model_patterns: list[list[str]] = [
+    ["gpt-4", "gpt-4", "gpt-3.5-turbo"],
+    ["gpt-4", "gpt-3.5-turbo", "gpt-4"],
+    ["gpt-4", "gpt-3.5-turbo", "togethercomputer/llama-2-70b-chat"],
+    ["gpt-4", "togethercomputer/llama-2-70b-chat", "gpt-3.5-turbo"],
+]
+
+
+def get_human_annotations(
+    target_model_patterns: list[list[str]],
+) -> list[AnnotationForEpisode]:
+    human_episodes: list[AnnotationForEpisode] = []
+    for pk in AnnotationForEpisode.all_pks():
+        episode_human = AnnotationForEpisode.get(pk)
+        episode_model = EpisodeLog.get(episode_human.episode)
+        if episode_model.models in target_model_patterns:
+            human_episodes.append(episode_human)
+    return human_episodes
+
+
+def get_dimension_correlation(
+    human_annotations: list[EpisodeLog],
+    machine_annotations: list[EpisodeLog],
+    dimension: str,
+) -> dict[str, float]:
+    # check the data is present
+    with_dimension_list = [
```

Comment:
This naming is confusing. Maybe 'episodes_with_valid_rewards'?

Reply:
okay!